packages/sdk-browser/src/__inputs__/public_api.swagger.json

```diff
@@ -7333,6 +7409,9 @@
         },
         "createReadWriteSessionResultV2": {
           "$ref": "#/definitions/v1CreateReadWriteSessionResultV2"
+        },
+        "deleteSubOrganizationResult": {
```

Comment:
Was deleteSubOrganization stuff supposed to be part of this PR? Or did it get autogenerated? Just flagging in case

Reply:
yeah that was part of the general release auto codegen; merged that in a [separate PR](https://github.com/tkhq/sdk/pull/379) and rebased this one; much smaller diff now